data/anaconda.conf.d/anaconda.conf

```diff
@@ -0,0 +1,34 @@
+# Anaconda configuration file.
+
+[Anaconda]
+
+# Run Anaconda in the debugging mode.
+debug = False
+
+[Services]
+
+# Enable SELinux usage in the installed system.
+# Valid values:
+#
+#  -1  The value is not set.
+#   0  SELinux is disabled.
+#   1  SELinux is enabled.
+#
+selinux = -1
+
+[Storage]
```

Comment:
Please add spaces only between major section. Otherwise it  is hard to keep track where is the major section starts and ends.

Reply:
Fixed.